### ISSUE TITLE
ast pretrained model use

### ISSUE THREAD

**Issue by syjunghwang** [lucidrains/musiclm-pytorch#21]
@lucidrains Hi, i want to use pretrained AST(https://github.com/YuanGongND/ast )model in this audiotransformer.
In musiclm-pytorch, the input shape of audio wave is 2 dim(ex. (2,1024)), however, in pretrained AST, we need a 3 dim input(ex. (batchsize, time, frequency)).

i saw a example which maded by someone for applying audiocap dataset to mulan training, but it didn't work because of difference of input dimension.
so, Do you have an idea how the existing dataset(ex. audiocap) to apply to this mulan model?
